### ISSUE TITLE
( is not a function in createFilter

### ISSUE THREAD

**Issue by MarkMurphy** [edy/redux-persist-transform-filter#10]
Think you've pushed a typo around here: https://github.com/edy/redux-persist-transform-filter/commit/a151e78c8f477e6daad666cfe2cbae4cd149f751#diff-090b05cdc4c33b5b4d778c13de44d8eaL30

I'm getting the following error in my react native project locally:

![screenshot 2017-06-13 20 07 49](https://user-images.githubusercontent.com/873474/27108520-09c2054a-5074-11e7-9d4a-98a83bb4ad09.png)

**Comment by MarkMurphy** [comment]
My bad, it wasn't that. The README is out of date. `createFilter` is now a default export.

**Comment by MarkMurphy** [comment]
Sorry, wrong again...I was reading the current master branches README. Not v0.0.10 which is correct.

**Comment by edy** [comment]
i will publish a new version later,  with an updated readme so that npmjs.com gets updated too

**Comment by Fenwil** [comment]
Hi @edy I'm having the same problem.
Is this already fixed with `0.0.11`? Or will be with a newer version?

**Nevermind like you said it was my import**
```javascript
import createFilter from 'redux-persist-transform-filter'
// I had this:
// import { createFilter } from 'redux-persist-transform-filter'
```
Thanks!

**Comment by edy** [comment]
i just pushed 0.0.12 with fixed default and named export. sorry for this one.
